docs/sources/developers/http_api/apis.md

```diff
@@ -0,0 +1,77 @@
+---
+aliases:
+  - ../../http_api/new_api_structure/
+  - ../../http_api/new_api_structure/
+canonical: /docs/grafana/latest/developers/http_api/new_api_structure/
+description: ''
+keywords:
+  - grafana
+  - http
+  - documentation
+  - api
+labels:
+  products:
+    - enterprise
+    - oss
+title: New API Structure
+---
+
+# Grafana's New API Structure
+
+## Overview
+
+Going forward, Grafana's HTTP API will follow a standardized API structure alongside consistent API versioning.
+
+## API Path Structure
+
+All Grafana APIs follow this standardized format:
+
+```
+/apis/<group>/<version>/namespaces/<namespace>/<kind>
```

Comment:
```suggestion
/apis/<group>/<version>/namespaces/<namespace>/<resource>
```

kind is usually the singular capitalized flavor... eg "Dashbaord" vs "dashboards"